lib/public/AppFramework/Utility/ITimeFactory.php

```diff
@@ -58,4 +58,12 @@ public function getDateTime(string $time = 'now', \DateTimeZone $timezone = null
 	 * @since 26.0.0
 	 */
 	public function withTimeZone(\DateTimeZone $timezone): static;
+
+	/**
+	 * @param string|null $timezone
+	 * @return \DateTimeZone
+	 * @throws \Exception
+	 * @since 28.0.0
+	 */
+	public function getTimezone(?string $timezone = null): \DateTimeZone;
```

Comment:
To be more inline with above method lets uppercase Z?

```suggestion
	public function getTimeZone(?string $timezone = null): \DateTimeZone;
```